backend/routes/auth.js

```diff
@@ -670,7 +670,11 @@ function signUp(req, res, next) {
 		&& Object.prototype.toString.call(req.body.lastName) === "[object String]"
 		&& Object.prototype.toString.call(req.body.countryCode) === "[object String]"
 		&& (!req.body.company || Object.prototype.toString.call(req.body.company) === "[object String]")
-		&& Object.prototype.toString.call(req.body.mailListAgreed) === "[object Boolean]") {
+		&& Object.prototype.toString.call(req.body.mailListAgreed) === "[object Boolean]"
```

Comment:
This is pretty old code XD since then we've added the type check functions in utils so we can avoid really long and clunkly code like this everywhere https://github.com/3drepo/3drepo.io/blob/master/backend/utils.js#L36

Can you change all of these to use `utils.isBool` `utils.isString`  etc (including the lines above your changes)

Reply:
doesn't look like you've done this one? or there's a commit you haven't pushed? @ChristosTsiotsias

Reply:
oh I commited but forgot to push -.-